main.js

```diff
@@ -367,7 +369,9 @@ function createWindow() {
       config.environment === 'test' ||
       config.environment === 'test-lib' ||
       config.environment === 'test-loki' ||
-      (mainWindow.readyForShutdown && windowState.shouldQuit())
+      config.environment.includes('test-integration')(
```

Comment:
```suggestion
      config.environment.includes('test-integration') || 
      (mainWindow.readyForShutdown && windowState.shouldQuit())
```
i think it's meant to be this

Reply:
fixed